src/main/java/com/jalasoft/webservice/model/OrcFile.java

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Execute CTRL + ALT + L in all java classes to auto-format according to the IDE rules